WIP: Update xCluster docs for DDL replication (#26395)
yugabyte/yugabyte-db: docs/content/preview/deploy/multi-dc/async-replication/async-transactional-failover.md

Determine if the estimated data loss is acceptable. If you know the exact time the Primary universe went down, you can compare it to the `safe_time`. Otherwise, use `safe_time_skew_sec` as rough measures to estimate the data loss.

For more information on replication metrics, refer to [xCluster metrics](../../../../launch-and-manage/monitor-and-alert/metrics/replication/).

### Restore the database(s) on B to a consistent state

Use PITR to restore the cluster to a consistent state that cuts off any partially replicated transactions. Use the xCluster `safe_time` obtained in the previous step as the restore time.

If there are multiple databases in the same cluster, do PITR on all the databases sequentially (one after the other).

<ul class="nav nav-tabs-alt nav-tabs-yb custom-tabs">
<li>
    <a href="#yugabyted-pitr" class="nav-link active" id="yugabyted-pitr-tab" data-bs-toggle="tab"
    role="tab" aria-controls="yugabyted-pitr" aria-selected="true">
    <img src="/icons/database.svg" alt="Server Icon">

>     Yugaybted

Reviewer: ```suggestion
    Yugabyted
```

    <i class="icon-shell"></i>
    Local
    </a>
</li>
</ul>
<div class="tab-content">
<div id="yugabyted-pitr" class="tab-pane fade show active" role="tabpanel" aria-labelledby="yugabyted-pitr-tab">

Restore the database to the `safe_time`:

```sh
./bin/yugabyted restore \
    --database <database> \
    --recover_to_point_in_time '<safe_time>'
```

</div>

<div id="local-pitr" class="tab-pane fade " role="tabpanel" aria-labelledby="local-pitr-tab">

1. Find the Snapshot Schedule name for the database:

```sh
./bin/yb-admin \
    -master_addresses <B_master_addresses> \
    list_snapshot_schedules
```

Expect output similar to the following:

```output.json
{
    "schedules": [
        {
            "id": "034bdaa2-56fa-44fb-a6da-40b1d9c22b49",
            "options": {
                "filter": "ysql.dr_db2",
                "interval": "1440 min",
                "retention": "10080 min"
            },
            "snapshots": [
                {
                    "id": "a83eca3e-e6a2-44f6-a9f2-f218d27f223c",
                    "snapshot_time": "2023-06-08 17:56:46.109379"
                }
            ]
        }
    ]
}
```

> 1. Restore to the `safe_time` of the database:

Reviewer: this numbering doesn't seem to render properly btw, all the bullets in the preview show 1.

Author: @ddhodge can you please help.  I could not figure out why this is broken

```sh
./bin/yb-admin \
    -master_addresses <B_master_addresses> \
    list_snapshot_restorations
```

Expect output similar to the following:

```output.json
{
    "restorations": [
        {
            "id": "a3fdc1c0-3e07-4607-91a7-1527b7b8d9ea",
            "snapshot_id": "3ecbfc16-e2a5-43a3-bf0d-82e04e72be65",
            "state": "RESTORED"
        }
    ]
}
```

</div>
</div>

### Delete the old replication group

```sh
./bin/yb-admin \
    -master_addresses <B_master_addresses> \
    delete_universe_replication <replication_group_id>
```

### Switch applications to B
Update the application connection strings to point to the new Primary universe (B).

After completing the preceding steps, the former Standby (B) is the new Primary (active) universe. There is no Standby until the former Primary (A) comes back up and is restored to the correct state.

## Set up reverse replication after A recovers

If the former Primary universe (A) doesn't come back and you end up creating a new cluster in place of A, follow the steps in [Setup transactional xCluster](../async-replication-transactional/).

> If universe A is brought back, to bring A into sync with B and set up replication in the opposite direction (B->A), the database on A needs to be dropped and recreated from a backup of B (Bootstrap). Before dropping the database on A, you can analyze it to determine the exact data that was lost by the failover.

Reviewer: ```suggestion
If universe A is brought back, to bring A into sync with B and set up replication in the opposite direction (B->A), the databases on A need to be dropped and recreated from a backup of B (Bootstrap). Before dropping the databases on A, you can analyze them to determine the exact data that was lost by the failover.
```

> Once you are ready to drop the database on A, follow the steps:

Reviewer: ```suggestion
Once you are ready to drop the databases on A, follow the steps:
```

    Automatic/Semi-automatic
    </a>
</li>
<li>
    <a href="#manual-drop" class="nav-link" id="manual-drop-tab" data-bs-toggle="tab"
    role="tab" aria-controls="manual-drop" aria-selected="false">
    Manual
    </a>
</li>
</ul>
<div class="tab-content">
<div id="dbscoped-drop" class="tab-pane fade show active" role="tabpanel" aria-labelledby="dbscoped-drop-tab">

Drop the replication group on A using the following command:

```sh
./bin/yb-admin \
    -master_addresses <A_master_addresses> \
    drop_xcluster_replication <replication_group_id>
```

You should see output similar to the following:

```output
Outbound xCluster Replication group rg1 deleted successfully
```

</div>

<div id="manual-drop" class="tab-pane fade " role="tabpanel" aria-labelledby="local-drop-tab">

- List the CDC streams on A to obtain the list of stream IDs as follows:

    ```sh
    ./bin/yb-admin \
    -master_addresses <A_master_ips> \
    list_cdc_streams
    ```

- For each stream ID, delete the corresponding CDC stream:

    ```sh
    ./bin/yb-admin \
    -master_addresses <A_master_ips> \
    delete_cdc_stream <streamID>
    ```

</div>
</div>

### Drop the database(s)

> Drop the database(s) on A

Reviewer: ```suggestion
Drop the database(s) on A.
```

### Perform setup from B to A

> Set up xCluster Replication from the Primary to standby universe (B to A) by following the steps in [Setup transactional xCluster](../async-replication-transactional/).

Reviewer: ```suggestion
Set up xCluster Replication from the Primary to Standby universe (B to A) by following the steps in [Setup transactional xCluster](../async-replication-transactional/).
```

}
```

2. Restore to the `safe_time` of the database:

> ```sh

Author: If I indent it then everything looks like an output
